### ISSUE TITLE
Can't save settings after install

### ISSUE THREAD

**Issue by mattgossman** [simonhamp/Gitamic#22]
**Describe the bug**
I installed Gitamic via Tools > Addons and after going to Settings > Gitamic and attempting to save the committer name and committer email address I get the error "Could not locate configuration for 'gitamic'." The logs show this:

`(Stillat\\Proteus\\Exceptions\\ConfigNotFoundException(code: 0): Could not locate configuration for 'gitamic'. at /Users/Matt/Sites/mysite/vendor/stillat/proteus/src/LaravelConfigWriter.php:341`

Attempting to access Gitamic from Tools > Gitamic in this unconfigured state produces another error "Gitonomy\Git\Exception\ProcessException"

By entering the settings in the .env file directly I was able to work around it.

**Versions:**
 - PHP: 8.0.2
 - Browser:  Safari
 - Statamic: 3.1.27
 - Gitamic release: 1.7.7
 - Gitamic version: Basic

**Comment by simonhamp** [comment]
@mattgossman does the file `config/gitamic.php` exist on the file system?

I believe the install script is supposed to create it, but perhaps it hasn't been able to for some reason - possibly folder permissions?

**Comment by mattgossman** [comment]
No, that file does not exist. I tried adding it to a new Statamic project and the config file wasn't added their either. No warnings or errors in the install process.

**Comment by simonhamp** [comment]
That's odd. I'll take a look into that.

In the meantime, could you try running the following command from the root of your project:

```bash
php artisan vendor:publish --provider=SimonHamp\\Gitamic\\ServiceProvider
```

Does this create the `config/gitamic.php` file?

**Comment by mattgossman** [comment]
Yes that moved the file successfully with this returned in the terminal window
`Copied File [/vendor/simonhamp/gitamic/config/gitamic.php] To [/config/gitamic.php]`

I looked at the Gitamic install process on a new statamic project and that line didn't appear:
`> @php artisan statamic:install --ansi
Discovered Addon: edalzell/forma
Discovered Addon: simonhamp/gitamic
Addon manifest generated successfully.
Copied Directory [/vendor/statamic/cms/resources/users] To [/resources/users]
Publishing complete.
Copied Directory [/vendor/statamic/cms/resources/dist] To [/public/vendor/statamic/cp]
Publishing complete.
Copied File [/vendor/simonhamp/gitamic/resources/dist/css/cp.css] To [/public/vendor/gitamic/css/cp.css]
Copied File [/vendor/simonhamp/gitamic/resources/dist/js/cp.js] To [/public/vendor/gitamic/js/cp.js]
Publishing complete.`

**Comment by simonhamp** [comment]
Note to self: according to [Statamic docs](https://statamic.dev/extending/addons#post-install-commands), `vendor:publish` should be run by default, but it seems that it may not have been.

Need to test to see if this has changed (i.e. been fixed) recently
